src/main/java/ch/wisv/areafiftylan/extras/rfid/model/RFIDLink.java

```diff
@@ -19,23 +19,30 @@
 
 import ch.wisv.areafiftylan.exception.InvalidRFIDException;
 import ch.wisv.areafiftylan.products.model.Ticket;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.hibernate.validator.constraints.Length;
 
 import javax.persistence.*;
 
 @Entity
+@Data
+@NoArgsConstructor
 public class RFIDLink {
+
     @Id
     @GeneratedValue
     private Long id;
 
+    @NonNull
+    @Length(min = RFID_CHAR_COUNT, max = RFID_CHAR_COUNT)
```

Comment:
As this is a Hibernate validator, it only gets checked if we call `@Validated` in a Controller somewhere. Is this the case? I think we only do that for the DTO. Anyway, is this covered by a test to make sure we're not getting a false sense of security here?